globals.d.ts

```diff
@@ -2,3 +2,85 @@ declare module '*.json' {
   const value: any
   export default value
 }
+
+export interface AlexaSmartHomeEvent {
```

Comment:
for locally developing types i would add a `types` folder to your project root, then make a directory called `aws-sdk` then in that dir a file called `index.d.ts` and in that file:

```
declare module 'aws-sdk' {
   ...add custom types
}
```

Then add the "typeRoots" property to your tsconfig:

```
typeRoots: [
  "node_modules/@types",
  "./types"
]
```